docs/aws.md

```diff
@@ -1,39 +1,213 @@
-  `mydomain.com`.
 
-* Pick a DNS name under this zone to be the name of your cluster.  kops will set up DNS so your cluster
-  can be reached on this name.  For example, if your zone was `mydomain.com`, a good name would be
-  `kubernetes.mydomain.com`, or `dev.k8s.mydomain.com`, or even `dev.k8s.myproject.mydomain.com`. We'll call this `NAME`.
+# Getting Started
 
-* Set `AWS_PROFILE` (if you need to select a profile for the AWS CLI to work)
+## Install kops
 
-* Pick an S3 bucket that you'll use to store your cluster configuration - this is called your state store.  You
-  can `export KOPS_STATE_STORE=s3://<mystatestorebucket>` and then kops will use this location by default.  We
-  suggest putting this in your bash profile or similar.  A single registry can hold multiple clusters, and it
-  can also be shared amongst your ops team (which is much easier than passing around kubecfg files!)
+From Homebrew:
 
-* Run "kops create cluster" to create your cluster configuration:
+```bash
+brew update && brew install --HEAD kops
 ```
-${GOPATH}/bin/kops create cluster --cloud=aws --zones=us-east-1c ${NAME}
+
+From Source:
+
+```bash
+go get -d k8s.io/kops
+cd ${GOPATH}/src/k8s.io/kops/
+git checkout release
+make
 ```
-(protip: the --cloud=aws argument is optional if the cloud can be inferred from the zones)
 
-* Run "kops update cluster" to build your cluster:
+See our [installation guide](build.md) for more information
+
+## Install kubectl
+
+It is a good idea to grab a fresh copy of `kubectl` now if you don't already have it.
+
+```
+brew install kubernetes-cli
 ```
-${GOPATH}/bin/kops update cluster ${NAME} --yes
+
+* [Kubernetes Latest Release](https://github.com/kubernetes/kubernetes/releases/latest)
+
+* [Installation Guide](http://kubernetes.io/docs/user-guide/prereqs/)
+
+
+```bash
+wget -O https://github.com/kubernetes/kubernetes/releases/download/v1.4.6/kubernetes.tar.gz
+sudo cp kubernetes/platforms/darwin/amd64/kubectl /usr/local/bin/kubectl
 ```
 
-If you have problems, please set `--v=8` and open an issue, and ping justinsb on slack!
+## Setup your environment
+
+### Setting up a kops IAM user
+
+
+In this example we will be using a dedicated IAM user to use with kops. This user will need basic API security credentials in order to use kops. Create the user and credentials using the AWS console. [More information](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/AWSCredentials.html).
+
+Kubernetes kops uses the official AWS Go SDK, so all we need to do here is set up your system to use the official AWS supported methods of registering security credentials defined [here](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials). Here is an example using the aws command line tool to set up your security credentials.
+
+```bash
+brew update && brew install awscli
+aws configure
+aws iam list-users
```

Comment:
I like

```console
$ aws iam list-users
```
Also, mention that then need to run configure, but don't so the output.  It is multi-line crap we don't want here.